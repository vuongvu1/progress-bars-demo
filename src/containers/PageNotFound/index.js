import React from 'react';
import styled from 'styled-components';

// #region Styles
const Wrapper = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;

  font-size: ${(props) => props.theme.fontSizeBig};
  font-weight: bold;
  height: 100%;
  color: ${(props) => props.theme.error};
`;
// #endregion Styles

const PageNotFound = () => (
  <Wrapper>
    <span role='img' aria-label='Cross'>
      ❌
    </span>{' '}
    Page Not Found
  </Wrapper>
);

export default PageNotFound;
