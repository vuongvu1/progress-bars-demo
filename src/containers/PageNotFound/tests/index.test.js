import React from 'react';
import { render } from '@testing-library/react';

import PageNotFound from '../index';

const renderComponent = (props = {}) => render(<PageNotFound {...props} />);

describe('<PageNotFound />', () => {
  it('should render <PageNotFound />', () => {
    const { container } = renderComponent();
    expect(container).toMatchSnapshot();
  });
});
