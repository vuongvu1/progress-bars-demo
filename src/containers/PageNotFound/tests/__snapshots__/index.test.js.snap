// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageNotFound /> should render <PageNotFound /> 1`] = `
<div>
  <div
    class="sc-bdVaJa iYIqAn"
  >
    <span
      aria-label="Cross"
      role="img"
    >
      ‚ùå
    </span>
     
    Page Not Found
  </div>
</div>
`;
